static/js/clients.js

```diff
@@ -0,0 +1,143 @@
+document.addEventListener('DOMContentLoaded', (event) => {
+    // Display filters only if javascript is enabled
+    let filtersBar = document.getElementById("filters");
+    filtersBar.style.display = "flex";
+
+    class Filter {
+        constructor(filterId, isAndFilter, allOf, anyOf) {
+            this.allOf = allOf;
+            this.anyOf = anyOf;
+            this.makeMenuInteractive(filterId);
+            this.enableFilters(filterId);
+            this.isAndFilter = isAndFilter;
+        }
+
+        makeMenuInteractive(filterId) {
+            let filterButton = document.getElementById(filterId)
+            filterButton.addEventListener("mouseup", (event) => {
```

Comment:
I think this should be click, not mouseup: https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event

Reply:
Fixed in 05ac9513